addon/bootstrap.js

```diff
@@ -51,8 +52,11 @@ const prefObserver = {
   }
 };
 
+let localePref = (prefs.getPrefType(LOCALE_PREF) && prefs.getCharPref(LOCALE_PREF)) || "en-US";
+
 function startup(data, reason) { // eslint-disable-line no-unused-vars
   console.log("bootstrap startup called");
+  Cu.import("chrome://screenshots/content/localizedTooltips.jsm");
```

Comment:
Does that work okay?  I saw some reference to needing to import non-system libraries in startup and then unload in shutdown.